src/nxdoc/nuxeo-add-ons/nuxeo-retention-management/nuxeo-retention-installation/nuxeo-retention-installation-strict.md

```diff
@@ -33,6 +33,10 @@ This is done adding the parameter `nuxeo.retention.strictmode.enabled=true` in t
 
 This must be done **prior to any usage of the Nuxeo instance** to guarantee the data consistency.
 
+{{#> callout type='info'}}
+Since LTS 2021 HF41, this property replaces and supersedes the property `nuxeo.retention.compliance.enabled`: if both are set, the value of nuxeo.retention.strictmode.enabled will be used.
```

Comment:
```suggestion
Since LTS 2021 HF41, this property replaces and supersedes the property `nuxeo.retention.compliance.enabled`: if both are set, the value of `nuxeo.retention.strictmode.enabled` will be used.
```